storage/src/cache/fscache/mod.rs

```diff
@@ -149,6 +149,36 @@ impl BlobCacheMgr for FsCacheMgr {
         self.get_or_create_cache_entry(blob_info)
             .map(|v| v as Arc<dyn BlobCache>)
     }
+
+    fn check_stat(&self) {
+        let guard = self.blobs.read().unwrap();
+        if guard.len() != self.blobs_need {
+            info!(
+                "blob mgr not ready to check stat, need blobs {} have blobs {}",
+                self.blobs_need,
+                guard.len()
+            );
+            return;
+        }
+
+        let mut all_ready = true;
+        for (_id, entry) in guard.iter() {
+            if !entry.is_all_data_ready() {
+                all_ready = false;
+                break;
+            }
+        }
+
+        if all_ready {
+            self.worker_mgr.stop();
+            let _ = self.metrics.data_all_ready.compare_exchange(
+                false,
```

Comment:
Why use `compare_exchange`, is it necessary? I suppose `store` it to True should meet the requirements here.

Reply:
store is enough , will fix it